pkg/managerclient/client.go

```diff
@@ -1,6 +1,6 @@
 // Copyright (C) 2017 ScyllaDB
```

Comment:
This commit has grown pretty big. Would you mind splitting it into
- changes to the client
- manual updates for the call sites
?